diff --git a/src/video.cpp b/src/video.cpp
index d168eda..66238ee 100755
--- a/src/video.cpp
+++ b/src/video.cpp
@@ -763,7 +764,7 @@ void core_video_refresh(const void *data, unsigned width, unsigned height, size_

         real_aspect_ratio = aspect_ratio;
         _351BlitRotation = isTate ? GO2_ROTATION_DEGREES_270 : GO2_ROTATION_DEGREES_0;
-        _351Rotation = isTate ? GO2_ROTATION_DEGREES_180 : GO2_ROTATION_DEGREES_270;
+        _351Rotation = isTate ? GO2_ROTATION_DEGREES_270 : GO2_ROTATION_DEGREES_0; // test
         first_video_refresh = false;
     }
     if (height != currentHeight || width != currentWidth)
